sections/multirow.liquid

```diff
@@ -0,0 +1,437 @@
+    {
+      "type": "select",
+      "id": "image_height",
+      "options": [
+        {
+          "value": "adapt",
+          "label": "t:sections.multirow.settings.image_height.options__1.label"
+        },
+        {
+          "value": "small",
+          "label": "t:sections.multirow.settings.image_height.options__2.label"
+        },
+        {
+          "value": "medium",
+          "label": "t:sections.multirow.settings.image_height.options__3.label"
+        },
+        {
+          "value": "large",
+          "label": "t:sections.multirow.settings.image_height.options__4.label"
+        }
+      ],
+      "default": "small",
+      "label": "t:sections.multirow.settings.image_height.label"
+    },
+    {
+      "type": "select",
+      "id": "desktop_image_width",
+      "options": [
+        {
+          "value": "small",
+          "label": "t:sections.multirow.settings.desktop_image_width.options__1.label"
+        },
+        {
+          "value": "medium",
+          "label": "t:sections.multirow.settings.desktop_image_width.options__2.label"
+        },
+        {
+          "value": "large",
+          "label": "t:sections.multirow.settings.desktop_image_width.options__3.label"
+        }
+      ],
+      "default": "medium",
+      "label": "t:sections.multirow.settings.desktop_image_width.label",
+      "info": "t:sections.multirow.settings.desktop_image_width.info"
+    },
+    {
+      "type": "select",
+      "id": "heading_size",
+      "options": [
+        {
+          "value": "h2",
+          "label": "t:sections.multirow.settings.heading_size.options__1.label"
+        },
+        {
+          "value": "h1",
+          "label": "t:sections.multirow.settings.heading_size.options__2.label"
+        },
+        {
+          "value": "h0",
+          "label": "t:sections.multirow.settings.heading_size.options__3.label"
+        }
+      ],
+      "default": "h1",
+      "label": "t:sections.multirow.settings.heading_size.label"
+    },
+    {
+      "type": "select",
+      "id": "text_style",
+      "options": [
+        {
+          "value": "body",
+          "label": "t:sections.multirow.settings.text_style.options__1.label"
+        },
+        {
+          "value": "subtitle",
+          "label": "t:sections.multirow.settings.text_style.options__2.label"
+        }
+      ],
+      "default": "body",
+      "label": "t:sections.multirow.settings.text_style.label"
+    },
+    {
+      "type": "select",
+      "id": "button_style",
+      "options": [
+        {
+          "value": "primary",
+          "label": "t:sections.multirow.settings.button_style.options__1.label"
+        },
+        {
+          "value": "secondary",
+          "label": "t:sections.multirow.settings.button_style.options__2.label"
+        }
+      ],
+      "default": "secondary",
+      "label": "t:sections.multirow.settings.button_style.label"
+    },
+    {
+      "type": "select",
+      "id": "desktop_content_position",
```

Comment:
I think we might need to add a similar setting for the image 🤔 Right now it stretches based on the length of the content. And it seem weird that it'd do that since we have a specific image height setting for it but it doesn't end up respecting it, in this scenario: 
<details>
<summary>Screenshot</summary>

![](https://screenshot.click/21-08-nc8sk-y363b.png)
</details>

Reply:
Yeah that could be something we do a follow up on if we see an actual need for it and if merchants do use longer texts.

Reply:
I agree, let's not change the behavior we reused from Image with text. We can correct it on both sections later on.